client/logic/game/effects/effect.gd

```diff
@@ -2,3 +2,12 @@ extends Node2D
 class_name Effect
 
 @onready var player: AnimationPlayer = $AnimationPlayer
+@onready var sprite_animation_2d: AnimatedSprite2D = $SpriteAnimation2D
+
+
+func _on_sprite_animation_2d_animation_looped() -> void:
```

Comment:
looped means restarted? there is no better signal? (I expect this comes from a signal) to destroy the effect?

Reply:
It's a signal yes. There is the signal finished in the sprite animation, but i don't get it to emit properly even when looped is turned off.